TTS/tts/models/base_tts.py

```diff
@@ -134,7 +134,7 @@ def get_aux_input_from_test_setences(self, sentence_info):
 
         # get speaker  id/d_vector
         speaker_id, d_vector, language_id = None, None, None
-        if hasattr(self, "speaker_manager"):
+        if self.speaker_manager is not None:
```

Comment:
I feel this is a bit unsafe because many modules set the attribute instead of defining it as none, I would stick to `hasattr` check first.